### PR TITLE
Update Fileversion for adapter and framework dlls

### PR DESCRIPTION
**Pull request by jayaranigarg**
https://github.com/Microsoft/testfx/issues/268

### CODE REVIEW

**Comment by jayaranigarg** [comment]
Verified on TestFx.CI.Real build definition.

File: scripts/Build.ps1 [hunk 1]
```diff
@@ -22,6 +22,14 @@ Param(
   [Alias("vs")]
   [System.String] $VersionSuffix = "dev",
   
+  [Parameter(Mandatory=$false)]
+  [Alias("fvp")]
+  [System.String] $FileVersionPrefix = "15.6",
```
Comment (smadala) [inline scripts/Build.ps1]: Nit: rename it to BuildVersionPrefix. and value as 14.0?

File: scripts/Build.ps1 [hunk 2]
```diff
@@ -22,6 +22,12 @@ Param(
   [Alias("vs")]
   [System.String] $VersionSuffix = "dev",
   
+  [Parameter(Mandatory=$false)]
+  [System.String] $BuildVersionPrefix  = "14.0",
+
+  [Parameter(Mandatory=$false)]
+  [System.String] $BuildVersionSuffix = "99.99",
```
Comment (singhsarab) [inline scripts/Build.ps1]: How does it look like in real when provided by the CI ? Share an example.

File: scripts/Build.ps1 [hunk 3]
```diff
@@ -74,6 +80,7 @@ $TFB_Configuration = $Configuration
 $TFB_FrameworkVersion = $FrameworkVersion
 $TFB_AdapterVersion = $AdapterVersion
 $TFB_VersionSuffix = $VersionSuffix
+$TFB_BuildVersion = if ($BuildVersionSuffix -ne '') { $BuildVersionPrefix + "." + $BuildVersionSuffix } else { $BuildVersionPrefix }
```
Comment (singhsarab) [inline scripts/Build.ps1]: not required.

File: scripts/Build.ps1 (hunk 2 above)
Comment (jayaranigarg) [reply scripts/Build.ps1]: looks like 1926.01

**Pull request merged**